CRATE.md

```diff
@@ -0,0 +1,148 @@
+<p align="center">
+  <img src="https://raw.githubusercontent.com/nats-io/nats.rs/master/logo/NATS-Rust.png">
+</p>
+
+<p align="center">
+    A <a href="https://www.rust-lang.org/">Rust</a> client for the <a href="https://nats.io">NATS messaging system</a>.
+</p>
+
+## Status
+[![License Apache 2](https://img.shields.io/badge/License-Apache2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
+[![Crates.io](https://img.shields.io/crates/v/nats.svg)](https://crates.io/crates/nats)
+[![Documentation](https://docs.rs/nats/badge.svg)](https://docs.rs/nats/)
+[![Build Status](https://travis-ci.org/nats-io/nats.rs.svg?branch=master)](http://travis-ci.org/nats-io/nats.rs)
+
+Warning: as of version 0.4.0, this crate has been rewritten, and several public
+APIs have changed. Please see the [docs](https://docs.rs/nats), the [new
+repo](https://github.com/nats-io/nats.rs), and [the
+changelog](https://github.com/nats-io/nats.rs/blob/master/CHANGELOG.md) for
+details.
```

Comment:
This is the initial text for the warning we talked about adding to this.

Reply:
I would add a note that the maintainers have changed and this is a new code base supported by Synadia, etc. Would make it a bit more detailed.